config/services.yaml

```diff
@@ -4,7 +4,7 @@
 # Put parameters here that don't need to change on each machine where the app is deployed
 # https://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
 parameters:
-    ilios_api_version: v2.0
+    ilios_api_version: v2.1
```

Comment:
is this really necessary? we've been making additive changes without bumping this version before.

Reply:
We always bump it for additions in order to match up the frontend version with the correct API.

Reply:
ah ok. i must have forgotten. thanks!